22:17 <@_rene_> vmiklos: hmm, I need to locally revert 
                ac631f9ebddbc3674f9628d941e81a7ee73c6c66. kfreebsd-* (yet) only 
                has java-gcj-compat/gcj-jdk available thus javadoc is de-facto 
                gjdoc and that doesn't know -notimestamp...
22:17 < IZBot> core - odk: use javadoc -notimestamp, so generated docs don't 
               unnecessarily differ after each build - 
http://cgit.freedesktop.org/libreoffice/core/commit/?id=ac631f9ebddbc3674f9628d941e81a7ee73c6c66
22:18 <@_rene_> .oO ( otoh, maybe I'll just implement a --without-docs option, 
                as they don't even need to be built when you only build the 
                binaries... )
22:21 <@_rene_> vmiklos: see 
https://buildd.debian.org/status/logs.php?pkg=libreoffice&ver=1%3A4.0.0-1&arch=kfreebsd-i386 :)

--- a/odk/pack/gendocu/makefile.mk-old	2013-02-08 22:21:38.581488115 +0100
+++ b/odk/pack/gendocu/makefile.mk	2013-02-08 22:21:53.449487676 +0100
@@ -59,7 +59,7 @@
         $(JAVA_SRC_DIR)$/com$/sun$/star$/beans$/OfficeConnection.java \
         $(JAVA_SRC_DIR)$/com$/sun$/star$/beans$/OfficeWindow.java
 
-JAVADOCPARAMS= -notimestamp -use -splitindex -windowtitle "Java UNO Runtime Reference" -header $(JAVADOCREFNAME) -d $(DESTDIRGENJAVAREF) -sourcepath $(JAVA_SRC_DIR) -classpath $(SOLARBINDIR)$/ridl.jar -linkoffline ../../common/ref ./uno -linkoffline http://java.sun.com/j2se/1.4.1/docs/api ./java $(JAVA_PACKAGES)
+JAVADOCPARAMS= -use -splitindex -windowtitle "Java UNO Runtime Reference" -header $(JAVADOCREFNAME) -d $(DESTDIRGENJAVAREF) -sourcepath $(JAVA_SRC_DIR) -classpath $(SOLARBINDIR)$/ridl.jar -linkoffline ../../common/ref ./uno -linkoffline http://java.sun.com/j2se/1.4.1/docs/api ./java $(JAVA_PACKAGES)
 
 JAVADOCLOG = $(MISC)$/javadoc_log.txt
 
